### PR TITLE
Backport trivial pr22

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
Script found more stuff this run :) Haven't reviewed, but it does compile

### CODE REVIEW

File: doc/fuzzing.md [hunk 1]
```diff
@@ -136,3 +136,15 @@ Full configure that was tested on macOS Catalina with `brew` installed `llvm`:
 ```
 ./configure --disable-ccache --enable-fuzz --with-sanitizers=fuzzer,address,undefined CC=/usr/local/opt/llvm/bin/clang CXX=/usr/local/opt/llvm/bin/clang++ --disable-asm
 ```
+
+# OSS-Fuzz
+
+Bitcoin Core participates in Google's [OSS-Fuzz](https://github.com/google/oss-fuzz/tree/master/projects/bitcoin-core)
```
Comment (UdjinM6) [inline doc/fuzzing.md:142]: 21856: not sure if this is actually relevant to us

File: doc/fuzzing.md [hunk 2]
```diff
@@ -140,11 +140,13 @@ Full configure that was tested on macOS Catalina with `brew` installed `llvm`:
 # OSS-Fuzz
 
 Bitcoin Core participates in Google's [OSS-Fuzz](https://github.com/google/oss-fuzz/tree/master/projects/bitcoin-core)
-program, which includes a dashboard of [publicly disclosed vulnerabilities](https://bugs.chromium.org/p/oss-fuzz/issues/list).
+program, which includes a dashboard of [publicly disclosed vulnerabilities](https://bugs.chromium.org/p/oss-fuzz/issues/list?q=bitcoin-core).
```
Comment (UdjinM6) [inline doc/fuzzing.md:143]: 21947: same

File: configure.ac [hunk 3]
```diff
@@ -411,7 +411,9 @@ if test "x$CXXFLAGS_overridden" = "xno"; then
   AX_CHECK_COMPILE_FLAG([-Wself-assign],[NOWARN_CXXFLAGS="$NOWARN_CXXFLAGS -Wno-self-assign"],,[[$CXXFLAG_WERROR]])
   AX_CHECK_COMPILE_FLAG([-Wunused-local-typedef],[NOWARN_CXXFLAGS="$NOWARN_CXXFLAGS -Wno-unused-local-typedef"],,[[$CXXFLAG_WERROR]])
   AX_CHECK_COMPILE_FLAG([-Wimplicit-fallthrough],[NOWARN_CXXFLAGS="$NOWARN_CXXFLAGS -Wno-implicit-fallthrough"],,[[$CXXFLAG_WERROR]])
-  AX_CHECK_COMPILE_FLAG([-Wdeprecated-copy],[NOWARN_CXXFLAGS="$NOWARN_CXXFLAGS -Wno-deprecated-copy"],,[[$CXXFLAG_WERROR]])
+  if test x$suppress_external_warnings != xyes ; then
```
Comment (UdjinM6) [inline configure.ac:414]: 22258: requires 18750

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
pls rename 9e4e412 to `Merge  bitcoin-core/gui#20: Wrap tooltips in the intro window` + see below

**Comment by PastaPastaPasta** [comment]
resolved all

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**